src/clj/game/cards/identities.clj

```diff
@@ -739,9 +739,13 @@
    "Weyland Consortium: Builder of Nations"
    {:implementation "Damage triggered manually"
     :abilities [{:label "Do 1 meat damage"
+                 :once :per-turn
+                 :prompt "Do a meat damage from identity ability?"
+                 :choices (cancellable ["Yes"])
                  :delayed-completion true
-                 :msg "do 1 meat damage"
-                 :effect (effect (damage eid :meat 1 {:card card}))}]}
+                 :effect (req (when (= target "Yes")
+                                (do (damage state :runner eid :meat 1 {:card card})
```

Comment:
Minor style thing: you don't need a `do` in the body of a `when`; the `do` is implicit, unlike `if`.